src/components/userService.ts

```diff
@@ -0,0 +1,47 @@
+import { Memento } from 'vscode';
+
+export class GlobalStateTokenStorage {
+	constructor(private readonly __globalState: Memento) {}
+
+	getUserToken() {
+		const clerkToken = this.__globalState.get('userToken');
```

Comment:
what kind of a user token that is? is it the proxy github token?

Reply:
we concluded that this is access token

Reply:
i renamed